src/java/CivilAirTransport.java

```diff
@@ -0,0 +1,33 @@
+package java;
+
+import java.util.Scanner;
+
+public class CivilAirTransport extends AirTransport {
+    int maksNumberOfPassengers;
+    boolean businessClass;
+
+    public CivilAirTransport(int power, int maksSpeed, int weight, String carModel, int wingSpan, int minLength,
+                             int maksNumberOfPassengers, boolean businessClass) {
+        super(power, maksSpeed, weight, carModel, wingSpan, minLength);
+        this.maksNumberOfPassengers=maksNumberOfPassengers;
+        this.businessClass=businessClass;
```

Comment:
@Hlieb Старайся ставить пробелы между операторами
```suggestion
        this.maksNumberOfPassengers = maksNumberOfPassengers;
        this.businessClass = businessClass;
```